stories/browser/src/UIAddressInput.tsx

```diff
@@ -2,7 +2,12 @@ import * as React from 'react';
 import { Platform, StyleSheet, TextInput } from 'react-native';
 
 import { ChatInputContainer, useChatInputValue } from '@tonlabs/uistory.chats';
-import { UITextView, useAutogrowTextView, UIInputAccessoryView } from '@tonlabs/uikit.inputs';
+import {
+    ActionButton,
```

Comment:
I also think that `ActionButton` should be moved to another package, but not in `uikit.input`. @savelichalex FYI

Reply:
@sertony ActionButton essentially the same as useClearButton, a little more independent but still only for input components, thats why I put it here. What place will be better?
May be it's better to made another independent component that can be clear button or action button. And then it may be placed in uikit.controls for example. For now everything, for no particular reason, lies in different places.

Reply:
I'm OK with leaving it in `uikit.input`